CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/ConversationContributionProvider.java

```diff
@@ -133,9 +134,11 @@ public final int getPriority() {
      */
     public static List<ConversationContributionProvider> getCompatibleProviders(GraphReadMethods graph) {
         List<ConversationContributionProvider> compatibleProviders = new ArrayList<>();
```

Comment:
I know you didn't add this but if you make this final please

Reply:
done